backend/routes/offline-payment.js

```diff
@@ -7,10 +7,7 @@ const { Network, Order } = require('../models')
 const { getConfig } = require('../utils/encryptedConfig')
 const { autoFulfillOrder } = require('../utils/printful')
 const makeOffer = require('./_makeOffer')
-const { OrderPaymentTypes, OrderPaymentStatuses } = require('../enums')
-
-const { getLogger } = require('../utils/logger')
-const log = getLogger('routes.offline-payment')
+const { OrderPaymentTypes } = require('../enums')
```

Comment:
We should keep the OrderPaymentStatuses and getLogger imports.
They are referenced in this file.

Reply:
Sorry, that's bad merge, that endpoint using both is moved to `routes/orders.js`